### ISSUE TITLE
Modify Error Function

### ISSUE THREAD

**Issue by Vedaevolution** [Microsoft/CNTK#1607]
Dear Community,

is there a way to access and modify an error function with python?
What i mean is, that normally you have to express an error function in cntk blocks. That's obviously due to backpropagation and gradient computation. The problem is that some error functions are complex and not that easy to express with such blocks.

I' playing around with RPN networks for object detection and for that it can be useful to measure the objectness with IOU. If you have a large input vector and many bounding boxes that's hard to do in cntk blocks. 

Is there a better way? The problem is that i need the results of one computation (one forward pass) to compute the IOU BEFORE the error is computed to do gradient descent.

Best Regards
Vedaevolution

**Comment by cha-zhang** [comment]
You can write custom error functions in Python such as this one: 
https://github.com/Microsoft/FERPlus

We are working on RPN. Hopefully it will be available in Examples/Image/Detection very soon.

**Comment by sayanpa** [comment]
Closing this issue since there was a suggestion and no further communication on this thread. Please reopen the issue with details if you need a follow up.
